boards/modalai/fc-v2/src/timer_config.cpp

```diff
@@ -0,0 +1,71 @@
+/****************************************************************************
+ *
+ *   Copyright (C) 2012 PX4 Development Team. All rights reserved.
```

Comment:
```suggestion
 *   Copyright (C) 2021 PX4 Development Team. All rights reserved.
```

Might as well set these.

Reply:
updated a few found in the fc-v2 board dir